packages/product-components/src/components/FeeRate/FeeRate.tsx

```diff
@@ -0,0 +1,18 @@
+import { NetworkType } from '@suite-common/wallet-config';
+import { getFeeUnits } from '@suite-common/wallet-utils';
+import { BigNumber } from '@trezor/utils';
+
+type FeeRateProps = {
+    feeRate: string | BigNumber;
+    networkType: NetworkType;
+};
+
+export const FeeRate = ({ feeRate, networkType }: FeeRateProps) => {
+    const fee = typeof feeRate === 'string' ? new BigNumber(feeRate) : feeRate;
+
+    return (
+        <span>
+            {fee.toFixed(2)}&nbsp;{getFeeUnits(networkType)}
```

Comment:
It is nice for Bitcoin but it does not make sense for EVM :( 
<img width="687" alt="Screenshot 2025-02-14 at 21 57 46" src="https://github.com/user-attachments/assets/4dbf3157-3eff-47b6-b809-c558d51040f5" />

Reply:
modified here #16702